web/sidebars.js

```diff
@@ -68,6 +68,7 @@ module.exports = {
           collapsed: true,
           items: [
             'auth/social-auth/overview',
+            'auth/social-auth/discord',
```

Comment:
The Discord auth provider shouldn't be the first one in the sidebar as it's not the most popular auth method we offer, let's put in the last place under the social auth group.